src/transactions/utils/validation.js

```diff
@@ -47,3 +47,19 @@ export const validateKeysgroup = keysgroup => {
 	}
 	return validatePublicKeys(keysgroup);
 };
+
+// Specification: https://github.com/LiskHQ/lisk/blob/development/logic/account.js#L373
+export const validateAddress = address => {
+	if (address.length < 2 || address.length > 22) {
```

Comment:
Not every address with length 22 is valid. A thorough check would see if it's in the valid range using bignum.